features.json

```diff
@@ -4,7 +4,7 @@
     "ember-routing-route-configured-query-params": null,
     "ember-libraries-isregistered": null,
     "ember-application-engines": true,
-    "ember-glimmer": null,
+    "ember-glimmer": true,
```

Comment:
For sure. We're just hard coding it for now, but will need a way to switch this for testing. 

Would we want to run the tests 2 times, once for HTMLBars and once for Glimmer?